packages/terra-date-picker/src/DatePickerField.jsx

```diff
@@ -217,16 +217,10 @@ const DatePickerField = (props) => {
 
   let ariaDescriptionIds;
 
-  if (help && error && isInvalid) {
+  if (error && isInvalid) {
```

Comment:
I think the help condition is needed if we are applying `${datePickerId}-help` to `ariaDescriptionIds`.

Reply:
We could probably break this out to two single checks like: 
```
    if (help) {
      ariaDescriptionIds = `${ariaDescriptionIds} ${datePickerId}-help`;
    }

    if (error && isInvalid) {
      ariaDescriptionIds = `${ariaDescriptionIds} ${datePickerId}-error`;
    }
```
not sure if that is simpler or not

Reply:
I think it should explicitly check for `help` if we're adding `${datePickerId}-help`.

Reply:
What my idea was to always provide `help` for `DatePickerField`, even if the user doesn't provide any help message still format will be there.

Reply:
That makes sense. Adding a comment here may help clarify the confusion.